shopfloor_single_product_transfer/services/single_product_transfer.py

```diff
@@ -325,29 +354,45 @@ def _use_handlers(self, handlers, *args, **kwargs):
             if response:
                 return response
 
+    def _add_location_package_lot_domain(
+        self, domain, location=None, package=None, lot=None
+    ):
+        if location:
+            domain = AND([domain, [("location_id", "=", location.id)]])
+        if package:
+            domain = AND([domain, [("package_id", "=", package.id)]])
```

Comment:
If no package, do not match a line with package
`domain = AND([domain, [("package_id", "=", package.id if package else False)]])`